open-metadata-implementation/access-services/glossary-view/glossary-view-client/src/main/java/org/odpi/openmetadata/accessservices/glossaryview/client/GlossaryViewClient.java

```diff
@@ -0,0 +1,599 @@
+    private static final String GET_ATTRIBUTES = PATH_ROOT + "/terms/{2}/attributes" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_SUBTYPES = PATH_ROOT + "/terms/{2}/subtypes" + PARAM_DELIMITER + PAGINATION;
+    private static final String GET_TYPES = PATH_ROOT + "/terms/{2}/types" + PARAM_DELIMITER + PAGINATION;
+
+    /**
+     * Create a new client that passes userId and password in each HTTP request.  This is the
+     * userId/password of the calling server.  The end user's userId is sent on each request.
+     *
+     * @param serverName name of the server to connect to
+     * @param serverPlatformRootURL the network address of the server running the OMAS REST servers
+     * @param userId caller's userId embedded in all HTTP requests
+     * @param password caller's userId embedded in all HTTP requests
+     *
+     * @throws InvalidParameterException null URL or server name
+     */
+    public GlossaryViewClient(String serverName, String serverPlatformRootURL, String userId, String password) throws
+            org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException {
+        super(serverName, serverPlatformRootURL, userId, password);
+    }
+
+    /**
+     * Extract all glossary definitions
+     *
+     * @param userId calling user
+     * @param serverName instance to call
+     *
+     * @return EntityDetailResponse glossaries
+     *
+     * @throws PropertyServerException if a problem occurs while serving the request
+     * @throws InvalidParameterException if parameter validation fails
+     * @throws GlossaryViewOmasException if a problem occurs on the omas backend
+     */
+    public GlossaryViewEntityDetailResponse getAllGlossaries(String serverName, String userId, Integer from, Integer size)
+            throws PropertyServerException, InvalidParameterException, GlossaryViewOmasException {
+
+        return getMultipleEntitiesPagedResponse("getAllGlossaries", GET_GLOSSARIES, serverName, userId, from, size);
+    }
+
+    /**
+     * Extract a glossary definition
+     *
+     * @param userId calling user
+     * @param serverName instance to call
+     * @param glossaryGUID glossary GUID
+     *
+     * @return EntityDetailResponse glossary
+     *
+     * @throws PropertyServerException if a problem occurs while serving the request
+     * @throws InvalidParameterException if parameter validation fails
+     * @throws GlossaryViewOmasException if a problem occurs on the omas backend
+     */
+    public GlossaryViewEntityDetailResponse getGlossary(String serverName, String userId, String glossaryGUID)
+            throws PropertyServerException, InvalidParameterException, GlossaryViewOmasException {
+
+        return getSingleEntityResponse("getGlossary", GET_GLOSSARY, serverName, userId, glossaryGUID);
+    }
+
+    /**
+     * Extract a term's home glossary
+     *
+     * @param userId calling user
+     * @param serverName instance to call
```

Comment:
ServerName should be in constructor - this applied to other calls.

Reply:
Removed serverName from method signatures and switched usage to field